client/src/components/AddEvent/AddEvent.js

```diff
@@ -0,0 +1,132 @@
+import { MuiPickersUtilsProvider, InlineDateTimePicker } from 'material-ui-pickers';
+import Geosuggest from 'react-geosuggest';
+
+class AddEvent extends React.Component {
+    state = {
+        available: true,
+        invite: false,
+        category: '',
+        selectedDate: new Date(),
+    };
+
+    handleDateChange = date => {
+        this.setState({ selectedDate: date });
+    };
+
+    handleChange = name => event => {
+        this.setState({ [name]: event.target.checked });
+    };
+
+    render() {
+        const { selectedDate } = this.state;
+
+        return (
+            <div className="add-event-page">
+                <form className="add-event-page-container">
+                    <label className="add-event-header">Please fill in all fields to create event</label>
+                    <TextField
+                        required
+                        className="add-event-text-field"
+                        label="Title"
+                        fullWidth
+                        autoComplete="off"
+                        inputProps={{ maxLength: 70 }}
+                    />
+
+                    <TextField
+                        required
+                        className="add-event-text-field"
+                        label="Description"
+                        fullWidth
+                        multiline
+                        autoComplete="off"
+                        inputProps={{ maxLength: 200 }}
+                    />
+
+                    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <InlineDateTimePicker
+                            label="Choose date and time"
+                            ampm={false}
+                            value={selectedDate}
+                            onChange={this.handleDateChange}
+                            className="add-event-picker"
+                        />
+                    </MuiPickersUtilsProvider>
+
+                    <FormGroup className="add-event-location">
+                        <FormControlLabel
+                            labelPlacement="top"
+                            label="Location"
+                            control={
+                                <Geosuggest />
+                            }
+                        />
+                    </FormGroup>
+
+                    <FormControl className="formControl">
+                        <InputLabel shrink htmlFor="age-native-label-placeholder">
+                            Category
+                        </InputLabel>
+                        <NativeSelect
+                            value={this.state.category}
+                            onChange={this.handleChange('category')}
+                            input={<Input name="category" />}
+                            className="add-event-select"
+                        >
+                            <option value="">none</option>
+                            <option value={10}>sport</option>
```

Comment:
Why  JSX, seems it can be just value="10"?

Reply:
It seems, that you are looking at one of the first commits. These code has been changed in the latest commits.